roles/manage-ssh-config-root-access/README.md

```diff
@@ -0,0 +1,48 @@
+manage_ssh_config_root_access role
+=========
+
+This role updates `/etc/sshd_config` to deny ssh password authentication for root.
+
+## Requirements
+This role needs to be executed by a user with 'become' privileges
+
+## Role Variables
+
+The following variables controls the functionality of `manage_ssh_config_root_access` role
+
+| Name | Purpose|Required|
+|---|---|---|
+|update_sshd|This variable when set to True will disable root's ability to login via SSH with a password|No|
+
+### Example Inventory: update /etc/ssh/sshd_config  
+
+```
+[all:vars]
+update_sshd_config: True
```

Comment:
I'd make this variable specific to the PermitRootLogin option - and set it to the value you'd like it to set, i.e.: `without-password`